src/main/java/com/prototype/booking/api/integration/RoomServiceGateway.java

```diff
@@ -0,0 +1,82 @@
+package com.prototype.booking.api.integration;
+
+import com.prototype.booking.api.MeetingRoomBooking;
+import com.prototype.booking.api.referencedata.MeetingRoom;
+import com.prototype.booking.api.referencedata.Timings;
+import com.prototype.booking.api.referencedata.TimingsRepository;
+import lombok.extern.slf4j.Slf4j;
+import lombok.var;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.annotation.Gateway;
+import org.springframework.integration.annotation.IntegrationComponentScan;
+import org.springframework.integration.annotation.MessagingGateway;
+import org.springframework.integration.config.EnableIntegration;
+import org.springframework.integration.dsl.IntegrationFlow;
+import org.springframework.integration.handler.LoggingHandler;
+import org.springframework.integration.jpa.dsl.Jpa;
+import org.springframework.integration.jpa.support.PersistMode;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+import java.util.Optional;
+
+@Configuration
+@EnableIntegration
```

Comment:
```suggestion
```
It's already present in **BookAMeetingRoomApplication**, is it required again?

Reply:
oops not required. Removed now